css/style.css

```diff
@@ -99,6 +99,15 @@ main {
   grid-area: download;
 }
 
+#download-simulations-btn {
+  box-shadow: 0 0 20px #ccc;
+  border-radius: 15px;
+  opacity: 0.6;
+}
+#download-simulations-btn:hover {
+  opacity: 1;
+}
+
```

Comment:
Please note that this button is already inside a `#download-button` div, so I'd call it like this instead:

`#download-button button {..}`
`#download-button button:hover {..}`

I hope it makes sense to you.

Additionally: when NOT on hover, the button seems to have a white overlay on top of it that mutes the button a little. Do you mind correcting the initial state to something brighter? Feel free to change button color as well (you can look at the `button` tag in the `css` to see where the yellow color is coming from.